src/bootstrap/src/core/config/config.rs

```diff
@@ -977,11 +1163,90 @@ impl Config {
             config.channel = channel;
```

Comment:
This override is applied after `config.channel` has already been read.. but it seems to be pre-existing, not sure if it was intended though. Let's leave it like it is for now.